lib/readline.js

```diff
@@ -24,7 +24,7 @@ exports.createInterface = function(input, output, completer, terminal) {
 };
 
 
-function Interface(input, output, completer, terminal) {
+function Interface(input, output, completer, terminal, historySize) {
```

Comment:
Gosh, this is so frustrating... Why do we need one more argument? Can't it be an options-only thing?